tabcmd/execution/logger_config.py

```diff
@@ -7,9 +7,13 @@
 def get_logger(name, logging_level):
     """function for logging statements to console and logfile"""
     logging_level = getattr(logging, logging_level.upper())
-    # TODO: in INFO level, we should leave out the (name) field
-    log_format = "%(levelname)s %(name)s  %(message)s"
-    logging.basicConfig(level=logging_level, format=log_format, filename="test.log", filemode="w")
+    if logging_level == 20:
```

Comment:
where's the number from?

Reply:
sweet fix, anyway

Reply:
Jac here are the levels https://docs.python.org/3/library/logging.html#levels

Reply:
Thanks, I just pulled the Magic number into a variable to make it more clear.